cabal-testsuite/src/Test/Cabal/Prelude.hs

```diff
@@ -603,16 +598,17 @@ withRepo repo_dir m = do
     withReaderT (\env' -> env' { testHaveRepo = True }) m
     -- TODO: Arguably should undo everything when we're done...
   where
-    repoUri env ="file+noindex://" ++ testRepoDir env
+    repoUri env ="file+noindex://" ++ (if isWindows
```

Comment:
For the reviewer: `file+noindex://C:\...` does not work but `file+noindex://C:/...` does